src/components/landing/WorkingSection.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Card from '@material-tailwind/react/Card';
+import CardImage from '@material-tailwind/react/CardImage';
+import CardBody from '@material-tailwind/react/CardBody';
+import Icon from '@material-tailwind/react/Icon';
+import H4 from '@material-tailwind/react/Heading4';
+import H6 from '@material-tailwind/react/Heading6';
+import LeadText from '@material-tailwind/react/LeadText';
+import Paragraph from '@material-tailwind/react/Paragraph';
+import StatusCard from 'components/landing/StatusCard';
+
+export default function WorkingSection() {
+    const cardImageSrc =
```

Comment:
Should we download these images and have them inside the product? This is useful if the owner of the image deletes it, right?